### ISSUE TITLE
How to make auto completion work with alias

### ISSUE THREAD

**Issue by lixiong-intelex** [mziyabo/PSKubectlCompletion#1]
Hi mziyabo,

Thanks for the great tool! I am new to k8s, so this helps me a lot!

I have a question as how to enable auto completion after I set alias for kubectl.

Specially, here is what I've in PS profile:
```
Import-Module PSKubectlCompletion
Register-KubectlCompletion
Set-Alias -Name k -Value kubectl -Scope global
```
Auto completion only works for kubectl but not for k.

Do you know how to make it work for alias?

Thanks very much!

**Comment by mziyabo** [comment]
Thanks Li, glad it's helping.

I just patched the module to v1.0.2 to get the alias scenario working. In your PS profile, `Set-Alias` should now come before Register-KubectlCompletion:

```
Import-Module PSKubectlCompletion
Set-Alias -Name k -Value kubectl -Scope global
Register-KubectlCompletion
```
In v1.0.2 the module will bind the completer to kubectl and it's aliases when you call Register-KubectlCompletion. 

Hopefully this helps but let me know if you have any trouble.

**Comment by lixiong-intelex** [comment]
Works like a charm! Thank you for your prompt reply! 

Take care!
